fstapi/build.rs

```diff
@@ -1,26 +1,70 @@
-use std::{env, path::PathBuf};
+use std::{env, path::{self, PathBuf}};
```

Comment:
```suggestion
use std::{env, path::PathBuf};
```

`path::self` can be removed.

Reply:
Thank you for your comments, I have addressed all of them (and tried clippy, let's see how the CI goes.)